### DISCUSSION TITLE
Unable to use fcitx5 for zh-hans input

### DISCUSSION THREAD

**Discussion by Dragon1573** [microsoft/WSL#9306] [Q&A]
## Summary

I want to use `fcitx5` for Simplified Chinese input in WSLg applications (such as `Tilix` and `microsoft-edge-stable`). But I failed to launch `fcitx5` with `fcitx5-configtool`. Logs shows that there's something wrong with the "dbus".

Is there any mistakes of my configurations? Or is there any walkaround to input CJK characters in WSLg?

## Screenshots

![image](https://user-images.githubusercontent.com/49941141/206903050-72eb17a8-21a6-4db9-8304-f292918d6ac5.png)

![image](https://user-images.githubusercontent.com/49941141/206903169-0ebbcd31-ab85-47f3-bddc-c326a1f429cf.png)

## Environments

- Windows 11 Professional 22H2
- Windows Subsystem Linux 2 (`systemd` enabled)
  - WSL Version: 1.0.3.0
  - Kernel Version: 5.15.79.1
  - WSLg Version: 1.0.47
  - MSRDC version: 1.2.3575
  - Direct3D Version: 1.606.4
  - DXCore Version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
  - Windows Version: 10.0.22621.819
- Ubuntu 22.10 Kinetic Kudu
- `fcitx5` 5.0.18

## Logs

[Informations from `fcitx5-diagnose` (zh-hans)](https://github.com/microsoft/WSL/files/10202185/fcitx5-diagnose.log)

**Comment by Dragon1573** [comment]
Wierd 😕 Try _Fcitx5_ 2 years after, this problem cannot be reproduced.

## Screenshots

![image](https://github.com/microsoft/WSL/assets/49941141/8d7d7387-f794-4b0a-af67-fe2ed4cadcf1)
![image](https://github.com/microsoft/WSL/assets/49941141/528edd37-d34e-4927-970f-b187f4fd51bb)
![image](https://github.com/microsoft/WSL/assets/49941141/3bc7dcf7-0bca-4440-a3f8-46d7b4e88bfd)

## Environments

```text
21:01:08 (base) D:\Repository 2ms pwsh> systeminfo.exe
OS 名称:          Microsoft Windows 11 专业版
OS 版本:          10.0.22635 暂缺 Build 22635
OS 配置:          独立工作站
OS 构建类型:      Multiprocessor Free
系统类型:         x64-based PC

21:02:26 (base) D:\Repository 2ms pwsh> wsl --version
WSL 版本： 2.2.2.0
内核版本： 5.15.150.1-2
WSLg 版本： 1.0.61
MSRDC 版本： 1.2.5105
Direct3D 版本： 1.611.1-81528511
DXCore 版本： 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows 版本： 10.0.22635.3430
```

```text
21:03:43 ~/Documents 0ms $ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 23.10
Release:        23.10
Codename:       mantic

21:04:11 ~/Documents 5ms $ fcitx5 --version
5.1.0

21:04:29 ~/Documents 10ms $ fcitx5-diagnose > /mnt/d/diagnose.log
```

## Logs

- [diagnose_before_launch.log](https://github.com/microsoft/WSL/files/14897272/diagnose.log)

**Comment by TTcheng** [comment]
According to your diagnose log before,  those environment variables is not effective
```
export GTK_IM_MODULE=fcitx
export QT_IM_MODULE=fcitx
export XMODIFIERS=@im=fcitx
```
